docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>This could be then broken into a single cell with a variable call for each. It'll render prettier in a notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>